packages/lib/zuauth/src/server.ts

```diff
@@ -47,8 +47,8 @@ function claimMatchesConfiguration(
   return (
     isEqualEdDSAPublicKey(claim.signer, config.publicKey) &&
     claim.partialTicket.eventId === config.eventId &&
-    !!config.productId &&
-    claim.partialTicket.productId === config.productId
+    (config.productId === undefined ||
```

Comment:
What's the semantics in PODBox if productID isn't defined?  The underlying PCD always has a productID, so is it meant to be ignored?  Is it set to be equal to the eventID?
This code implements the former (productID will be ignored) which seems fine.  We could be extra-strict and require that the productID not be revealed when the config doesn't specify it, but that seems optional